tests/test_flight.py

```diff
@@ -0,0 +1,43 @@
+"""
+Copyright (c) 2010-present by Jaxl Innovations Private Limited.
+All rights reserved.
+Redistribution and use in source and binary forms,
+with or without modification, is strictly prohibited.
+"""
+from webhooks.flight import (
+JaxlIVRFlightWebhook,
+MAIN_MENU_PROMPT,
+get_service_prompt,
+confirm_service,
+)
+from jaxl.ivr.frontend.base import BaseJaxlIVRWebhookTestCase
+class TestJaxlIVRFlightWebhook(BaseJaxlIVRWebhookTestCase):
+"""Test cases for JaxlIVRFlightWebhook IVR."""
+webhook_klass = JaxlIVRFlightWebhook
+def setUp(self) -> None:
+super().setUp()
+self.start_call()
+def test_main_menu(self) -> None:
+"""Test main menu prompt is returned as expected after start of call."""

```

Comment:
Can you please fix formatting, some how all indentation is gone, this is now no longer a valid Python code